core_backend/app/routers/manage_content.py

```diff
@@ -48,7 +49,10 @@ async def create_content(
     """
 
     payload = _create_payload_for_qdrant_upsert(
-        content.content_title, content.content_text, content.content_metadata
+        content.content_title,
+        content.content_text,
+        content.content_metadata,
+        content_language=content.content_language,
```

Comment:
Can you either add all variables names or remove `content_language=` for consistency? I always prefer writing the var names cos even if the order changes in a refactor the calls will still work